elide-core/src/main/java/com/yahoo/elide/core/security/checks/DatastoreEvalFilterExpressionCheck.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.core.security.checks;
+
+import com.yahoo.elide.core.security.ChangeSpec;
+import com.yahoo.elide.core.security.RequestScope;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Optional;
+
+public class DatastoreEvalFilterExpressionCheck<T> extends OperationCheck<T> {
+    @Getter @Setter
+    private boolean executedInMemory;
```

Comment:
executedInMemory is past tense which makes it sounds like this has already happened.  I believe the intent here it to mark whether this expression _should_ run in memory when evaluated. 

I would change this to 'executeInMemory'.